cdc/owner/changefeed.go

```diff
@@ -211,7 +211,7 @@ func (c *changefeed) tick(ctx cdcContext.Context, state *orchestrator.Changefeed
 		// This condition implies that the DDL resolved-ts has not yet reached checkpointTs,
 		// which implies that it would be premature to schedule tables or to update status.
 		// So we return here.
-		log.Info("barrierTs < checkpointTs, premature to schedule tables or update status",
+		log.Debug("barrierTs < checkpointTs, premature to schedule tables or update status",
```

Comment:
Why use Debug level? Is it verbose?

Reply:
yes , it's quite verbose.

Reply:
it would flood logs files, which makes trouble shooting hard.